.github/workflows/ci.yml

```diff
@@ -100,19 +100,19 @@ jobs:
           build-args: ${{ matrix.build-args }}
           context: ${{ matrix.context }}/
           cache-from: |
-            type=registry,ref=${{ steps.login.outputs.registry }}/firezone/cache/${{ matrix.image_name }}:${{ env.CACHE_TAG }}
-            type=registry,ref=${{ steps.login.outputs.registry }}/firezone/cache/${{ matrix.image_name }}:main
+            type=registry,ref=${{ steps.login.outputs.registry }}/firezone-staging-cache/${{ matrix.image_name }}:${{ env.CACHE_TAG }}
```

Comment:
This will allow us to be more aggressive with cache removal and avoid mistakes of removing containers that we still need by accident (because one of its layers can be just untagged artifact references from a tagged one).